tests/render/test_column_section_renderer.py

```diff
@@ -1273,8 +1247,6 @@ def test_ValidationResultsColumnSectionRenderer_render_header_evr_with_unescaped
         validation_results=[evr_with_unescaped_dollar_sign],
     )
 
-    print(content_block.to_json_dict())
```

Comment:
`non-blocking`
You missed adding this one back.

Reply:
added back :)